src/extension/extensionServer.ts

```diff
@@ -240,8 +240,15 @@ export class ExtensionServer implements vscode.Disposable {
                         return mobilePlatform.runApp();
                     })
                     .then(() => {
+                        if (request.arguments.type === "reactnativedirect") {
+                            generator.step("mobilePlatform.disableJSDebuggingMode");
+                            generator.add("isDirect", true, false);
+                            this.logger.info(localize("DisableJSDebugging", "Disable JS Debugging"));
```

Comment:
I think receiving "Disable JS Debugging" message might be confusing when the debugger is starting. It would be better if we do this message more clear for understanding.